src/DynamicData/Cache/SourceCache.cs

```diff
@@ -23,15 +23,16 @@ public class SourceCache<TObject, TKey> : ISourceCache<TObject, TKey>
         /// Initializes a new instance of the <see cref="SourceCache{TObject, TKey}"/> class.
         /// </summary>
         /// <param name="keySelector">The key selector.</param>
+        /// <param name="keyEqualityComparer">The comparer to use to determine if two keys are equivalent</param>
```

Comment:
Just one request though, can you add a full stop, and also just document what comparer is used if null.